### ISSUE TITLE
When the list of options is open or closed, VoiceOver does not announce it.

### ISSUE THREAD

**Issue by claudiulodro** [woocommerce/selectWoo#6]
In VoiceOver on Safari when you open or close the dropdown you do not hear any sort of audible indicator that the dropdown has opened or closed. In Chrome you do, but most people use VoiceOver with Safari.
